modules/@apostrophecms/piece-type/ui/apos/components/AposImageRelationshipEditor.vue

```diff
@@ -90,24 +101,155 @@ export default {
       modalTitle: {
         key: 'apostrophe:editImageRelationshipTitle',
         title: this.title
-      }
+      },
+      groups: [],
+      currentTab: null,
+      tabs: [],
+      fields: {},
+      visibleSchema: [],
+      alignedFields: [ 'width', 'height' ]
     };
   },
   async mounted() {
     this.modal.active = true;
     this.docReady = true;
+
+    this.setVisibleSchema();
+    this.setNestedSchema();
+
+    // this.setGroups();
+    // this.setTabs();
+    // this.setFields();
   },
   methods: {
     async submit() {
       this.$emit('modal-result', this.docFields.data);
       this.modal.showModal = false;
     },
-    updateDocFields(value) {
-      this.docFields = value;
+    updateDocFields(value, repopulateFields = false) {
+      if (value.hasErrors) {
+        return;
+      }
+
+      this.docFields.data = {
+        ...this.docFields.data,
+        ...value.data,
+        ...repopulateFields && { _id: cuid() }
```

Comment:
This ID is needed for `AposSchema` to re populate the fields with the new values. We pass it only when dcoFields is updated when cropping the image, we do not repopulate fields when updating size from inputs.